egs/wsj/s5/steps/nnet3/train_dnn.py

```diff
@@ -555,6 +578,17 @@ def Train(args, run_opts):
     # egs_dir will be updated if there is realignment
     cur_egs_dir=egs_dir
 
+    # If true, compute the dropout proportion for training.
+    # and write dropout proportion.
+    dp_prop = [1.0] * num_iters
+    if add_ephemeral_connection:
+      init_zero_dp_iter = 10 + num_hidden_layers * args.add_layers_period
```

Comment:
Maybe we could add a arg naming "dropout_schedule" to make the changing of dropout rate more easily.